index.html

```diff
@@ -0,0 +1,99 @@
+                </p>
+                <ol>
+                    <li>At vero et odio </li>
+                    <li>eos et</li>
+                    <li>accusamus</li>
+                    <li>iusto</li>
+                    <li>dignissimos</li>
+                    <li>survived</li>
+                </ol>
+                <p>
+                    It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+                </p>
+            </section>
+            <section class="main-section" id="Where_can_I_get_some">
+                <header class="h2">Where can I get some?</header>
+                <code>Where can I get some?</code>
+                <p>
+                    There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+                </p>
+                <p>
+                    It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+                </p>
+            </section>
+            <section class="main-section" id="standard_Lorem_Ipsum">
+                <header class="h2">The standard Lorem Ipsum passage, used since the 1500s</header>
+                <code>The standard Lorem Ipsum passage, used since the 1500s<code>
+                <p>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+                <p>
+                <p>
+                    It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+                </p>
+            <section>
```

Comment:
Close this  tag

Reply:
Remember to be polite and kind to the student! I would suggest this as an alternative:
> https://github.com/guillainbisimwa/Review-past-project/pull/2/files/7817154ac709020410317bab47e8d663ea51c7df#r739686511

And remember to be more specific with the line where you would like the changes to be made! if you select a large area is hard for the student to know what piece of code you are referring to. Check my example above 🔼